tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+"compilerOptions": {
+  "esModuleInterop": true,
+  "jsx": "react",
```

Comment:
change it react-jsx, you will need to refactor some code but it will allow to use key property in your components

Reply:
fixed